### ISSUE TITLE
Remove last row from editabletreeview example. 

### ISSUE THREAD

**Issue by xoac** [baoboa/pyqt5#16]
Hi,
I have tried to run [editabletreeview example](https://github.com/baoboa/pyqt5/tree/master/examples/itemviews/editabletreemodel). and want to remove last row. Application crash with error:

```
Traceback (most recent call last):
  File "editabletreemodel.py", line 178, in index
    childItem = parentItem.child(row)
  File "editabletreemodel.py", line 60, in child
    return self.childItems[row]
IndexError: list index out of range
[1]    31900 abort (core dumped)  python3 editabletreemodel.py
```

It's looks like it removes quick good. But next wannt to select index that don't exist.

Here is my work around in class `TreeItem` I change def of `child(self, row)` to:

```
    def child(self, row):
        if row < len(self.childItems):
            return self.childItems[row]
       # else return  None
```

I am bigener at qt and pyqt so not sure this change is OK in other cases, if yes please give me know I will do my firs merge request :)

**Comment by xoac** [comment]
Ok I made pull request #17

**Comment by baoboa** [comment]
closed with request #17
